From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Outfit7TSB <sunnybunnies09@naver.com>
Date: Sun, 16 Feb 2025 23:28:32 +0900
Subject: [PATCH] Add Foundation API

Foundation has a better version of javaplugin method than Spigot.
NOTE: ALL PLUGINS WILL BREAK! USE WITH CAUTION

Foundation By Kangarko, Copyright (C) Mineacademy.org

#SpigotLiberation #Boycottmd5 #ImpeachMichealDardis #BuiltByBit

diff --git a/build.gradle.kts b/build.gradle.kts
index c0d5c1ed35049a1034a465498a9a8598a18ebb93..a0726849232081ac482bb01b37d4b4d6abd360fa 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -24,8 +24,17 @@ abstract class MockitoAgentProvider : CommandLineArgumentProvider {
 }
 // Paper end - configure mockito agent that is needed in newer java versions
 
+ repositories {
+ 
+    maven {
+        url = uri("https://jitpack.io")
+    }
+ 
+ }
+
 dependencies {
     implementation(project(":NogyangSpigot-API")) // 버킷-스피곳-페이퍼-레인보우 통합 API
+	implementation("com.github.kangarko:Foundation:6.9.17") // 버킷 API 부트스트랩(신)
     implementation("ca.spottedleaf:concurrentutil:0.0.3") // Paper - Add ConcurrentUtil dependency
     // Paper start
     implementation("org.jline:jline-terminal-ffm:3.27.1") // use ffm on java 22+
